packages/web/src/features/languages/ImportLanguageGlossesView.tsx

```diff
@@ -185,17 +185,22 @@ export default function ImportLanguageGlossesView() {
                         <FormLabel htmlFor="import">
                           {t('languages:import_language').toUpperCase()}
                         </FormLabel>
-                        <ComboboxInput
-                          id="import"
+                        <Controller
                           name="import"
-                          className="w-full"
-                          autoComplete="off"
-                          required
-                          aria-describedby="import-error"
-                          items={importLanguages.data.map((language) => ({
-                            label: language,
-                            value: language,
-                          }))}
+                          rules={{ required: true }}
+                          render={({ field }) => (
+                            <ComboboxInput
+                              {...field}
+                              id="import"
+                              className="w-full"
+                              autoComplete="off"
+                              aria-describedby="import-error"
+                              items={importLanguages.data.map((language) => ({
+                                label: language,
+                                value: language,
+                              }))}
+                            />
+                          )}
                         />
```

Comment:
Here is what it looks like to use Controller with a complex input. Before this was all hidden inside the ComboboxInput, but now the connection is made explicit.